terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -1244,6 +1244,15 @@ def remove_inconsequential_statements(statements: list[JSON]) -> list[JSON]:
                             - /var/run/docker.sock:/var/run/docker.sock
                           labels:
                             io.rancher.os.scope: "user"
+                        prune-images:
+                          image: docker:19.03.15-dind
```

Comment:
There is another occurrence of this version in this file. Please extract a variable.